base/stats.go

```diff
@@ -106,6 +107,7 @@ func (g *GlobalStat) initResourceUtilizationStats() {
 		SystemMemoryTotal:                   NewIntStat(ResourceUtilizationSubsystem, "system_memory_total", nil, nil, prometheus.GaugeValue, 0),
 		WarnCount:                           NewIntStat(ResourceUtilizationSubsystem, "warn_count", nil, nil, prometheus.CounterValue, 0),
 		CpuPercentUtil:                      NewFloatStat(ResourceUtilizationSubsystem, "process_cpu_percent_utilization", nil, nil, prometheus.GaugeValue, 0),
+		Uptime:                              NewDurStat(ResourceUtilizationSubsystem, "uptime", nil, nil, prometheus.GaugeValue, time.Now()),
```

Comment:
This can be a counter rather than a gauge. Duration should only ever go upwards in our use case